### PR TITLE
Backport test.sh to RELEASE-3.1.X for CI job

### PR DESCRIPTION
**Pull request by epwn**
Our Ursula_Test_Ceph_Swift_Ubuntu_Package_Branch_Weekly now depends on this script being present in Ursula.

### CODE REVIEW

**Comment by nirajdp76** [comment]
LGTM

File: test.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+if [[ -z ${PACKAGE_VERSION} && "${INSTALL_METHOD}" == "package" ]]; then
+    echo "openstack_package_version is not specified!"
+    exit 1
+fi
+
+export STACK_RC=~/jenkins.stackrc
+export CI=jenkins
+export PATH=${WORKSPACE}/venv/bin:$PATH
+export TEST_ENV=${TEST_ENV:-ci-full-ubuntu}
```
Comment (zfeldstein) [inline test.sh]: Should this be ci-full ?

Comment (epwn) [inline test.sh]: nope

**Comment by zfeldstein** [comment]
LGTM

**Pull request merged**